rakefile

```diff
@@ -9,6 +11,56 @@ include PerfResultsHelper
 
 REF_ARCH_STD, REF_ARCH_LARGE = ["S","L"]
 
+# Create a trumped up class to get a Beaker helpers outside of a Beaker run
```

Comment:
In RAS we used a `beaker_helper` to consolidate beaker-related functionality for the acceptance tests. Would it make sense to do something similar here for this functionality?

https://github.com/puppetlabs/ref_arch_setup/blob/904f5126b30223a90181bfd047e808cfffbefb85/Rakefile#L5

Reply:
I don't think so.  The purpose of this class is not to introduce any new functionality, it is simply to access beaker helpers in the Rakefile.  It is very context specific here.